install_files/securedrop-ossec-agent/DEBIAN/postinst

```diff
@@ -18,6 +18,8 @@ set -e
 
 case "$1" in
     configure)
+        # Disable do-release-upgrade notification
+        sed -i 's/Prompt=.*/Prompt=never/' /etc/update-manager/release-upgrades
```

Comment:
Just a quick observation.. i think it would make more sense to drop this in `securedrop-config` since that package already goes on both servers. thoughts @redshiftzero ?

Reply:
Ahhh, yep - that is probably better

Reply:
:+1:  k i can make that change locally and test against it and commit when all is well

Reply:
:+1: